apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/VedtaksvurderingService.kt

```diff
@@ -68,7 +68,15 @@ class VedtaksvurderingService(
 
     fun lagreIverksattVedtak(behandlingId: UUID) {
         repository.hentVedtak(behandlingId)?.also {
+            val iverksattTidspunkt = LocalDateTime.now()
```

Comment:
Mulig dette tidspunktet burde kommet tilbake fra utbetaling? Det er jo mulig for all slags delay på kafka, slik at iverksatt tidspunkt kan potensielt være langt tilbake i tid. Kanskje ikke så farlig for statistikk sin del?

Reply:
Det er et godt poeng. Jeg lager en Jira på det, for å få dette fikset først :D